docs/release-notes.md

```diff
@@ -1,5 +1,7 @@
 # Release Notes
 
+* [Percona Distribution for PostgreSQL 12.10 Second Update](release-notes-v12.10.upd.md)
```

Comment:
I think this should be "release-notes-v12.10.upd2.md"

Reply:
Fixed.